index.js

```diff
@@ -630,9 +644,9 @@ app.post('/api/sensorEvent', Validator.body(['coreid', 'event']).exists(), async
       const coreId = request.body.coreid
       const sensorEvent = request.body.event
       if (sensorEvent === SENSOR_EVENT.DURATION) {
-        alertType = ALERT_REASON.DURATION
+        alertType = ALERT_TYPE.SENSOR_DURATION
       } else if (sensorEvent === SENSOR_EVENT.STILLNESS) {
```

Comment:
Also on a schema note - should we update the firmware to send NO_MOTION (and consequently the Sensor Event enum? will it then become redundant?

Reply:
We could do. But it would need to be the whole string `SENSOR_DURATION` and `SENSOR_NO_MOTION` (or `SENSOR_STILLNESS` after I change it in `brave-alert-lib`).